src/tests/integration/test_mssql.py

```diff
@@ -1,4 +1,3 @@
-import pyodbc
```

Comment:
Why was this removed from here and placed into the function?

Reply:
because this allows us to run a subset of the integration tests. e.g., if I'm only interested in running postgres tests, then I shouldn't need to install pyodbc, since that's only required for mssql tests. it removes the complexity of requiring all dependencies, some of this are problematic to install